app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
I do actually prefer slim (and coffeescript), but for the sake of familiarity I chose to use ERB